### PR TITLE
Add deprecation notice for launch templates. 

### PR DESCRIPTION
**Pull request by bharath-123**
As discussed in office hours, we will be working on deprecating launch configurations in favour of launch templates. Add the deprecation notice to the release notes of the 1.19 and 1.20 release.

### CODE REVIEW

File: docs/releases/1.19-NOTES.md [hunk 1]
```diff
@@ -144,6 +144,8 @@ has been updated by a newer version of kOps unless it is given the `--allow-kops
 
 * The experimental node authorizor is now ignored if you are using kubernetes 1.19. The feature will be removed in 1.20. Worker nodes will instead be authorized using kops-controller.
 
+* Support for AWS LaunchConfiguration is being deprecated and will be removed in kOps 1.21.
```
Comment (hakman) [inline docs/releases/1.19-NOTES.md:147]:
```suggestion
* Support for AWS LaunchConfiguration has been deprecated and will be removed in kOps 1.21.
```

File: docs/releases/1.20-NOTES.md [hunk 2]
```diff
@@ -50,6 +50,8 @@
 
 * Due to lack of maintainers, the Aliyun/Alibaba Cloud support has been deprecated. The current implementation will be left as-is until the implementation needs updates or otherwise becomes incompatible. At that point, it will be removed. We very much welcome anyone willing to contribute to this cloud provider.
 
+* Support for AWS LaunchConfiguration is being deprecated and will be removed in kOps 1.21.
```
Comment (hakman) [inline docs/releases/1.20-NOTES.md:53]:
```suggestion
* Support for AWS LaunchConfiguration has been deprecated and will be removed in kOps 1.21.
```

**Review by hakman — CHANGES_REQUESTED** [review]

File: docs/releases/1.20-NOTES.md (hunk 2 above)
Comment (bharath-123) [reply docs/releases/1.20-NOTES.md:53]: Thanks! Will make these changes and push!

**Comment by bharath-123** [comment]
Should we add a log line in the code warning of the deprecation too?

**Comment by bharath-123** [comment]
https://github.com/kubernetes/kops/pull/10809 I have added it to the 1.19 and 1.20 release notes. Will add it to the 1.21 release notes.

**Pull request merged**